admin/js/boldgrid-backup-admin-folder-exclude.js

```diff
@@ -400,9 +402,13 @@ BoldGrid.FolderExclude = function( $ ) {
 		$( '#configure_folder_exclude' ).on( 'click', self.onClickConfigure );
 
 		self.toggleStatus();
-		self.toggleConfig();
+		$type.each( function() {
+			self.toggleConfig( this );
+		} );
 
-		$type.on( 'change', self.onChangeType );
+		$type.on( 'change', function() {
+			self.onChangeType( this );
```

Comment:
Do you actually need to make this change? `this` is already passed / available in the `onChangeType` method.

Reply:
Not quite.....If you don't change it, the variable passed is the event object, wheras in this situation, the variable passed is the element that changed.

![image](https://user-images.githubusercontent.com/49331357/80002720-fe4dfb00-848d-11ea-9bf1-4871bc5cb241.png)